public/app/features/datasources/components/EditDataSource.tsx

```diff
@@ -106,9 +107,14 @@ export function EditDataSourceView({
   const { readOnly, hasWriteRights, hasDeleteRights } = dataSourceRights;
   const hasDataSource = dataSource.id > 0;
 
+  const ds = getDataSourceSrv()?.getInstanceSettings(dataSource.uid);
+
+  const hasAlertingEnabled = Boolean(ds?.meta?.alerting ?? false);
+  const isAlertManagerDatasource = ds?.type === 'alertmanager';
+  const alertingSupported = hasAlertingEnabled || isAlertManagerDatasource;
```

Comment:
I'm not sure we want to show the badge for Alertmanager Datasources since it's implied? Wdyt?

Reply:
Yes, but maybe we should show it for consistency?

Reply:
Yea I don't have any particular opinions on this one, showing it probably makes sense 👍